Dockerfile.ros1

```diff
@@ -1,44 +1,31 @@
 ARG ROS_DISTRIBUTION=noetic
-FROM ros:$ROS_DISTRIBUTION-ros-core
+FROM ros:$ROS_DISTRIBUTION-ros-base
```

Comment:
What extra packages are needed from base?

Reply:
Primarily `rosdep`. If `rosdep` is already installed, we do not have to manually decide if we have to install `python-rosdep` (melodic) or `python3-rosdep` (noetic)